CMakeLists.txt

```diff
@@ -1350,6 +1352,25 @@ if(MSVC)
   )
 endif()
 
+if(WIN32)
+  find_file(QT_TRANSLATION_FILE qt_de.qm PATH_SUFFIXES share/qt5/translations)
+  get_filename_component(QT_TRANSLATIONS ${QT_TRANSLATION_FILE} DIRECTORY)
+  install(
+    DIRECTORY "${QT_TRANSLATIONS}"
+    DESTINATION "${MIXXX_INSTALL_DATADIR}"
+    # QT 5 translations have been separated into several files, and most of the qt_xx.qm files
+    # contain just shortcuts to load the qtbase, qtmultimedia etc files.
```

Comment:
Please add a conclusion that leads to the regex.

Reply:
What regex? There is no regex here...

Reply:
The regex is unchanged from before...